test/stat/stat_test.go

```diff
@@ -53,6 +53,15 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 	}
 	prometheusPod := pods[0]
 
+	controllerPods, err := TestHelper.GetPodsForDeployment(TestHelper.GetConduitNamespace(), "controller")
```

Comment:
Super minor, but you could reuse the `pods` and `err` variables from above, avoiding the needs for the new `controllerPods` var. I'd rewrite this as:

```
	pods, err = TestHelper.GetPodsForDeployment(TestHelper.GetConduitNamespace(), "controller")
	if err != nil {
		t.Fatalf("Failed to get pods for controller: %s", err)
	}
	if len(pods) != 1 {
		t.Fatalf("Expected 1 pod for controller, got %d", len(pods))
	}
	controllerPod := pods[0]
```